### ISSUE TITLE
PLE2502 should detect U+061C ARABIC LETTER MARK

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#20058]
### Summary

[`bidirectional-unicode` (PLE2502)](https://docs.astral.sh/ruff/rules/bidirectional-unicode/) should detect U+061C ARABIC LETTER MARK, a bidi control character similar to U+200F RIGHT-TO-LEFT MARK.
```
$ python -c 'print("""example = "x\N{ARABIC LETTER MARK}" * 100  #    "\N{ARABIC LETTER MARK}x" is assigned\nprint(len(example))""")' >ple2502.py

$ cat ple2502.py
example = "x؜" * 100  #    "؜x" is assigned
print(len(example))

$ python ple2502.py
200

$ ruff --isolated check ple2502.py --select PLE2502
All checks passed!

### Version

ruff 0.12.10 (c68ff8d90 2025-08-21)
